packages/server/src/auth.ts

```diff
@@ -45,6 +50,18 @@ export class ServerAuth {
     return authorized === did
   }
 
+  verifyAdmin(req: express.Request): boolean {
+    const header = req.headers.authorization || ''
+    if (!header.startsWith(BASIC)) return false
+    const token = header.slice(BASIC.length)
+    const [username, password] = uint8arrays
+      .toString(uint8arrays.fromString(token, 'base64url'), 'utf8')
```

Comment:
I believe basic auth typically calls for "regular" base64 encoding rather than url-safe base64.  Just a nit, certainly not too hung-up on it :)

Reply:
ah you're right. & i think padded too. used to using base64url elsewhere in our code haha